resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -257,6 +304,11 @@ public void operationComplete(ChannelFuture future) throws Exception {
         });
     }
 
+    // Only here to override in unit tests.
+    int dnsRedirectPort(@SuppressWarnings("unused") InetAddress server) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "server". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1172)